Library/Formula/kettle.rb

```diff
@@ -0,0 +1,98 @@
+class Kettle < Formula
+  homepage "http://community.pentaho.com/projects/data-integration/"
+  url "https://downloads.sourceforge.net/project/pentaho/Data%20Integration/5.0.1-stable/pdi-ce-5.0.1-stable.zip"
+  sha1 "c34fa3dbe8b75280fd3f7ddcaf609acbcdd2ed78"
+
+  def install
+    rm_rf Dir["*.{bat}"]
+    libexec.install Dir["*"]
+
+    (var + "log/kettle").mkpath
+    (etc + "kettle/simple-jndi").mkpath
+
+    # We don't assume that kitchen and pan are in anyway unique command names so we'll prepend "pdi"
+    %w[kitchen pan].each do |command|
+      wrapper_file = libexec + command
+      wrapper_file.write command_wrapper_file_content(command)
+      chmod 0755, wrapper_file
+      bin.install_symlink wrapper_file => "pdi#{command}"
+    end
+
+    carte_password_config_file = etc + "kettle/pwd/kettle.pwd"
+    carte_password_config_file.write carte_password_config_content unless carte_password_config_file.exist?
+
+    carte_server_config_file = etc + "kettle/carte.xml"
+    carte_server_config_file.write carte_server_config_content unless carte_server_config_file.exist?
```

Comment:
Simplified by using `(etc+"path").install` as suggested and using example files packaged with Kettle thus no need to `write` file content.